compendium/DeclarativeServices/test/TestReferenceManagerImpl.cpp

```diff
@@ -170,10 +201,15 @@ TEST_P(ReferenceManagerImplTest, TestIsSatisfied)
     auto fakeMetadata = GetParam();
     ReferenceManagerImpl refManager(fakeMetadata,
                                     GetFramework().GetBundleContext(),
-                                    fakeLogger, FakeComponentConfigName);
-    EXPECT_EQ(refManager.IsSatisfied(), (refManager.IsOptional() ? true : false)) << "Initial state is SATISFIED only if cardinality is optional";
-    auto reg = bc.RegisterService<dummy::Reference1>(std::make_shared<dummy::Reference1>());
-    EXPECT_EQ(refManager.IsSatisfied(), true) << "State expected to be SATISFIED after service registration";
+                                    fakeLogger,
+                                    FakeComponentConfigName);
+    EXPECT_EQ(refManager.IsSatisfied(),
+              (refManager.IsOptional() ? true : false))
```

Comment:
Can this be simplified to:
EXPECT_EQ(refManager.IsSatisfied(), refManager.IsOptional())
?

Reply:
it can.